test/plugin/timezone.test.js

```diff
@@ -308,7 +308,7 @@ describe('CustomPraseFormat', () => {
 
 describe('startOf and endOf', () => {
   it('corrects for timezone offset in startOf', () => {
-    const originalDay = dayjs.tz('2010-01-01 00:00:00', NY)
```

Comment:
Do you have any thoughts why it was being failed with this date?

Reply:
The test failed because dayjs's startOf('day') recalculates the start of the day in local time, which can cause timezone offset changes. On '2010-01-01', the timestamp was adjusted due to the way dayjs handles timezone shifts, leading to a mismatch.